test/verify/check-storage-stratis

```diff
@@ -0,0 +1,370 @@
+        self.devices_dropdown("Create Stratis pool")
+        self.dialog_wait_open()
+        self.dialog_wait_val("name", "pool1")
+        self.dialog_cancel()
+        self.dialog_wait_close()
+
+        b.click('.sidepanel-row:contains("pool0")')
+        b.wait_visible('#storage-detail')
+
+        # Create two filesystems
+        b.click("button:contains(Create new Filesystem)")
+        self.dialog_wait_open()
+        self.dialog_set_val('name', 'fsys1')
+        self.dialog_set_val('mount_point', '/run/fsys1')
+        b.assert_pixels("#dialog", "create-fsys")
+        self.dialog_apply()
+        self.dialog_wait_close()
+
+        b.wait_in_text("#detail-content", "fsys1")
+        b.assert_pixels("#detail-content", "fsys-row")
+
+        b.click("button:contains(Create new Filesystem)")
+        self.dialog({'name': 'fsys2',
+                     'mount_point': '/run/fsys2'})
+        b.wait_in_text("#detail-content", "fsys2")
+
+        # Check that they have entries in fstab
+        self.assertNotEqual(m.execute("grep /run/fsys1 /etc/fstab"), "")
+        self.assertNotEqual(m.execute("grep /run/fsys2 /etc/fstab"), "")
+
+        # Rename one filesystem
+        self.content_dropdown_action(1, "Rename")
+        self.dialog({'name': "fsys1-renamed"})
+        b.wait_in_text("#detail-content", "fsys1-renamed")
+
+        # Destroy one filesystem
+        self.content_dropdown_action(1, "Delete")
+        self.dialog_wait_open()
+        b.assert_pixels("#dialog", "delete-fsys")
+        self.dialog_apply()
+        self.dialog_wait_close()
+        b.wait_not_in_text("#detail-content", "fsys1-renamed")
+
+        # Unmount and remount the other filesystem
+        self.content_dropdown_action(1, "Unmount")
+        self.content_tab_wait_in_info(1, 1, "Mount point", "The filesystem is not mounted")
+        self.content_row_action(1, "Mount")
+        self.dialog({})
+        self.wait_mounted(1, 1)
+
+        # Make a copy of the filesystem
+        self.content_dropdown_action(1, "Copy")
+        self.dialog_wait_open()
+        self.dialog_set_val('name', 'fsys2-copy')
+        self.dialog_set_val('mount_point', '/run/fsys2-copy')
+        b.assert_pixels("#dialog", "copy-fsys")
+        self.dialog_apply()
+        self.dialog_wait_close()
+        b.wait_in_text("#detail-content", "fsys2-copy")
+
+        # Delete the copy
+        self.content_dropdown_action(2, "Delete")
+        self.confirm()
+        b.wait_not_in_text("#detail-content", "fsys2-copy")
+
+        # Add a data blockdev
+        b.click('#detail-sidebar .pf-c-card__actions button')
+        self.dialog_wait_open()
+        self.dialog_set_val('disks', {dev_3: True})
+        b.assert_pixels("#dialog", "add-disk")
+        self.dialog_apply()
+        self.dialog_wait_close()
+
+        b.wait_in_text('#detail-sidebar', dev_3)
+        b.wait_in_text('#detail-sidebar .sidepanel-row:contains(%s)' % dev_3, "data")
+
+        # Add a cache blockdev
```

Comment:
After reading this, I finally realized what "Tier" means, and that there are "cache" block devs. (These are not visible on initial creation).

Can you please test adding a data block dev after adding a cache one? This reproducibly hangs for me in Fedora 34 at least, without any journal output, I waited about a minute until I force-reloaded the page. This might be a naughty, but at least an upstream bug report.

Reply:
Interesting!

Reply:
This also hangs for me, and also on the command line.

Reply:
I have filed https://github.com/stratis-storage/stratisd/issues/2789, I think that is enough, we don't need to have a test ourselves.